community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactions.java

```diff
@@ -266,6 +281,39 @@ private void assertDatabaseIsRunning()
     @Override
     public KernelTransactionsSnapshot get()
     {
-        return new KernelTransactionsSnapshot( allTransactions );
+        return new KernelTransactionsSnapshot( allTransactions, Clock.SYSTEM_CLOCK.currentTimeMillis() );
```

Comment:
We are using `Clocks` in a couple of places in here. Would it make sense to have it passed in?